asn1tools/rvcs.py

```
@@ -106,28 +106,103 @@ def cmd_test(args):
 
         print("%s is %dB in UPER" % (path, len(uper)))
 
+def schema_path_dict(schemas_list):
+    schema_dict = {}
+    for schema in schemas_list:
+        path_elements = schema.split (os.path.sep)
+        if ".asn" in path_elements[len(path_elements)-1]:
```

```suggestion
        if path_elements[-1].endswith(".asn"):
```

---

Why check for the .asn extension? Are we silently ignoring some files the user specified?